src/wpantund/NCPInstanceBase-Addresses.cpp

```diff
@@ -175,14 +175,16 @@ NCPInstanceBase::on_mesh_prefix_flags_to_string(uint8_t flags, bool detailed)
 			break;
 		}
 
-		snprintf(c_string, sizeof(c_string), "flags:0x%02x [on-mesh:%s def-route:%s config:%s dhcp:%s slaac:%s pref:%s prio:%s]",
+		snprintf(c_string, sizeof(c_string), "flags:0x%02x [on-mesh:%s def-route:%s config:%s dhcp:%s slaac:%s pref:%s nd-dns:%s dp:%s prio:%s]",
```

Comment:
@XPAWAN I noticed that this change is not passing CI test.

Seems like it's due to this change here which impacts the output of `wpanctl` command which is used by `toranj/wpan.py` to parse and get the flags:
https://github.com/openthread/openthread/blob/f57c5e23709c33d6b0c1a4e209223f35fd040529/tests/toranj/wpan.py#L1176-L1204

------

I don't see any way to update this in `wpantund` without breaking the compatibility with current `wpan.py` implementation.

Here is what I can think of and suggest as way to move forward:
- In a PR in OT repo, we update the `wpan.py` to allow extra chars in between `pref:` and `prio:` (and just skips over it without parsing the content)
- We would then need to merge that first and the change here from this PR in `wpantund` should pass and we can merge this PR.
- After that we can later consider updating the `wpan.py` to actually parse the new flags.

Any other ideas?

Reply:
@XPAWAN I submitted https://github.com/openthread/openthread/pull/6240 which relaxes the parsing of the prefix list `wpanctl` output in `troanj` (what I mentioned above).

However, I noticed even with this change the test were still failing (on removing of prefix) -> https://github.com/abtink/openthread/runs/2043151764?check_suite_focus=true

I think there is some other missing piece here in this PR in `wpantund` itself.
- This PR is changing the DBUS commenad (adding new flags to it).
- The `tool-cmd-add-prefix.c` is updated to use the new DBUS message format, but the `tool-cmd-remove-prefix` is still using old DBUS format (without new flags). 
- On the `wpantund` side the PR is parsing DBUs message expecting the new flags to be present and so the `remove-prefix` is failing